src/test/txvalidationcache_tests.cpp

```diff
@@ -114,7 +114,7 @@ BOOST_FIXTURE_TEST_CASE(tx_mempool_block_doublespend, TestChain100Setup)
 // should fail.
 // Capture this interaction with the upgraded_nop argument: set it when evaluating
 // any script flag that is implemented as an upgraded NOP code.
-static void ValidateCheckInputsForAllFlags(const CTransaction &tx, uint32_t failing_flags, bool add_to_cache)
+static void ValidateCheckInputsForAllFlags(const CTransaction &tx, uint32_t failing_flags, bool add_to_cache) EXCLUSIVE_LOCKS_REQUIRED(cs_main)
```

Comment:
15413 seems incomplete

Reply:
I've looked through 15413 and i can't seem to find what would be missing.

Reply:
I think I was referring to the bench/* changes. Which means we should probably mark this as partial, or drop until block_assemble and duplicate_inputs exsists

Reply:
Sounds good